### PR TITLE
chore: Update `README.md` after migration to the JS SDK monorepo

### PR DESCRIPTION
**Pull request by Lms24**
This replaces the previous (now stale) Readme content with a note that the project was moved to the JS SDK monorepo. Additionally, we're adding some links to the migration guide, the "new" readme and to the monorepo.

(Happy to bring back the old content if y'all think it should stay here but just FYI it's not up to date anymore after we introduced the breaking changes)

### CODE REVIEW

**Review by billyvg — APPROVED** [review]
Should we archive this repo too?

File: README.md [hunk 1]
```diff
@@ -1,152 +1,10 @@
-# sentry-replay
+# Sentry Replay
 
-Note: Session Replay is currently in beta.
+## Attention: This project was moved to the [getsentry/javascript](https://github.com/getsentry/sentry-javascript/tree/master/packages/replay) monorepo.
```
Comment (mydea) [inline README.md]:
```suggestion
## Attention: This project was moved to the [getsentry/sentry-javascript](https://github.com/getsentry/sentry-javascript/tree/master/packages/replay) monorepo.
```

Comment (Lms24) [reply README.md]: ahh good catch, thanks :)

**Comment by Lms24** [comment]
>Should we archive this repo too?

Generally I'd vote yes but should we maybe give it a few weeks in case anyone is still visiting this repo?

**Review by mydea — APPROVED** [review]

**Review by jas-kas — APPROVED** [review]

**Comment by jas-kas** [comment]
> Generally I'd vote yes but should we maybe give it a few weeks in case anyone is still visiting this repo?

Agree w/ Lukas here!

**Pull request merged**